### ISSUE TITLE
❌ 🐞 Bug Report: PHP Warning:  Trying to access array offset on value of type bool in elementor\includes\managers\image.php::108

### ISSUE THREAD

**Issue by Litalino** [elementor/elementor#20707]
## Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

I edit the link : ..wp-admin/post.php?post=2611&action=elementor

got an error: PHP Warning:  Trying to access array offset on value of type bool in C:\laragon\www\foxiz\wp-content\plugins\elementor\includes\managers\image.php on line 108

Current line 108:  $urls[ $size ] = wp_get_attachment_image_src( $id, $size )[0];

`<html>
<body>

PHP: showing 1 of 1PHP: 2022-12-10 23:12:09 [warning X 22][C:\laragon\www\foxiz\wp-content\plugins\elementor\includes\managers\image.php::108] Trying to access array offset on value of type bool [array (   'trace' => ' #0: Elementor\Core\Logger\Manager -> shutdown() ', )]
--

</body>
</html>`

### Steps to reproduce

`<html>
<body>
<div class="elementor-system-info-section elementor-system-info-plugins" style="outline: 0px; margin-bottom: 10px; color: rgb(60, 67, 74); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Roboto, Oxygen-Sans, Ubuntu, Cantarell, &quot;Helvetica Neue&quot;, sans-serif; font-size: 13px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(240, 240, 241); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">

Active Plugins |   |  
-- | -- | --
Autoptimize - 3.1.4 | By Frank Goossens (futtta) |  
Breadcrumb NavXT - 7.1.0 | By John Havlik |  
Contact Form 7 - 5.7 | By Takayuki Miyoshi |  
Elementor - 3.9.0 | By Elementor.com |  
Elementor Pro - 3.9.0 | By Elementor.com |  
Envato Market - 2.0.7 | By Envato |  
Foxiz Core - 1.6.8 | By Theme-Ruby |  
HubSpot All-In-One Marketing - Forms, Popups, Live Chat - 9.2.12 | By HubSpot |  
MC4WP: Mailchimp for WordPress - 4.8.12 | By ibericode |  
Post Views Counter - 1.3.12 | By Digital Factory |  
Similar post-title checker - 1.0.0 | By WP-Parsi team |  
Wordpress Keyword Tool - 2.3.3 | By Atef |  
WP Content Crawler - 1.12.0 | By Turgut Sarıçam |  
WP Super Cache - 1.9.1 | By Automattic |  

</div>
</body>
</html>`

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

`<html>
<body>

Server Environment |   |  
-- | -- | --
Operating System: | WINNT |  
Software: | nginx/1.22.0 |  
MySQL version: | MySQL Community Server - GPL v8.0.30 |  
PHP Version: | 8.1.10 |  
PHP Memory Limit: | 2048M |  
PHP Max Input Vars: | 3000 |  
PHP Max Post Size: | 2G |  
GD Installed: | Yes |  
ZIP Installed: | Yes |  
Write Permissions: | All right |  
Elementor Library: | Connected

</body>
</html>`

`<html>
<body>

Version: | 6.1.1 |  
-- | -- | --
Site URL: | https://foxiz.dtv |  
Home URL: | https://foxiz.dtv |  
WP Multisite: | No |  
Max Upload Size: | 2 GB |  
Memory limit: | 40M |  
Max Memory limit: | 2048M |  
Permalink Structure: | /%year%/%monthnum%/%day%/%postname%.html |  
Language: | en-US |  
Timezone: | 7 |  
Admin Email: | info@demo.dtv |  
Debug Mode: | Inactive

</body>
</html>` #

**Comment by nicholaszein** [comment]
Hello @Litalino, 

Thank you for submitting your issue! 🙏 

### ❌ While reviewing your report, we noticed it was missing some key pieces of information.

- **Clear and concise title:** Please provide a clear and concise title that gives context to your issue;
- **Prerequisites:** please search the repository before submitting an issue;
- **Description:** In your description, you need to explain in detail what happened, as you would explain to someone that doesn't know anything about your website or Elementor. This is important information that will help solve your issue quicker;
- **Steps to replicate:** Think of it as explaining to someone over the phone how to replicate your issue. They can't see the problem, so explain it in detail in a way that we can try to replicate your issue in our own environments;
- **Isolating the Problem:** You need to make sure this only happens while using the latest versions of Elementor, Elementor Pro, and the Hello Elementor theme. We do not provide support for 3rd-party themes and plugins.
- **System info:** Please include the [System Info](https://elementor.com/help/systems-info/) as outlined in our [guidelines](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml) report available in your WordPress Dashboard > Elementor > System Info. This information is crucial for us to create an exact copy of your environment and validate your issue. Watch [this video](https://www.youtube.com/watch?v=tDXdynsnYuA) to learn how to download the report. We can't accept issues without it.
- **One issue per thread:** We can't track multiple issues per thread. Open a new issue for each subject.

Without them, we simply can't help. 😕 

👉 Please follow the guidelines for opening a new [🐞 Bug Report](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml), so we can help you better.


### ⚠️ Note that GitHub issues are reserved for Elementor 🐞 Bug Reports only, and should not be used for support.

❔ If you have questions or need general support, please use one of the following channels:

- [Elementor Global Community](https://facebook.com/groups/Elementors/) on Facebook
- [Elementor Help Center](https://docs.elementor.com)
- [Support Forum](https://wordpress.org/support/plugin/elementor) on WordPress.org 
- [Elementor Developers Documentation](https://developers.elementor.com/)

💎 If you have purchased [Elementor Pro](https://elementor.com/pro), you are entitled to personal support, please see your purchase email or visit your [Elementor Account page](https://my.elementor.com/) for details.


#### For  💡 **New Ideas** or 🚀 **Feature Requests** please [create a new Discussion](https://github.com/elementor/elementor/discussions/new) in the [Discussions](https://github.com/elementor/elementor/discussions) tab.

Best regards
